code/obj/item/stun_baton.dm

```diff
@@ -279,7 +279,8 @@ TYPEINFO(/obj/item/baton)
 					if (user.mind && user.mind.special_role == ROLE_VAMPTHRALL && isvampire(M) && user.is_mentally_dominated_by(M))
 						boutput(user, "<span class='alert'>You cannot harm your master!</span>")
 						return
-					if (M.do_dodge(user, src))
+					if (M.do_dodge(user, src) || M.parry_or_dodge(user, src))
+						boutput(user, "<span class='alert'>[M] dodges your blow!</span>")
```

Comment:
seems redundant since both of those procs also generate chat messages

Reply:
Fair point